tar/multitape/multitape_write.c

```diff
@@ -715,6 +716,9 @@ flushtape(TAPE_W * d, int isapart, int extrastats)
 	struct tapemetaindex tmi;
 	char * tapename;
 
+	/* We need a tapename. */
```

Comment:
I'd add "(anonymous dry runs are assigned a fake name before this point)" to avoid confusion about that.